config/initializers/sentry.rb

```diff
@@ -0,0 +1,12 @@
+Sentry.init do |config|
+  config.dsn = 'https://e43549eae9bb4f81891972b87e79b5ee@o1209445.ingest.sentry.io/6356423'
+  config.breadcrumbs_logger = [:sentry_logger, :http_logger]
+
+  # To activate performance monitoring, set one of these options.
+  # We recommend adjusting the value in production:
+  config.traces_sample_rate = 0.5
+  # or
+  config.traces_sampler = lambda do |context|
+    0.5
+  end
```

Comment:
2 things here.
- Do we need both of these or just one?
- I think 0.5 is quite high. It means we will send traces for every other transaction. On previous projects ive gone for 0.01? Ive asked to see if we can get free sentry premium. If so lets keep this number nice and high, otherwise, I think we might have to drop it a bit?

Reply:
Sounds good, I'll move it to 0.01 for the time being